data/models.py

```diff
@@ -0,0 +1,194 @@
+from django.contrib.gis.db import models
+from django.core.validators import MaxValueValidator, MinValueValidator
+from django.db.models.functions import Now
+
+from dags.sources.config.shared_constants import (
+    SUGGESTION_ATRAITER,
+    SUGGESTION_AVALIDER,
```

Comment:
```suggestion
    SUGGESTION_A_TRAITER,
    SUGGESTION_A_VALIDER,
```

non bloquant, à voir ensemble